content/library_handles.tex

```diff
@@ -25,8 +25,8 @@
 Handle of type \CTYPE{shmem\_team\_t} that corresponds to a team of \acp{PE}
 that share a memory domain. When this handle is used by some \ac{PE},
```

Comment:
As long as we're editing this text -- :neckbeard: -- Do we need the `When this handle is used by some \ac{PE},` bit? Seems like unnecessary text, we could instead simplify to `SHMEM_TEAM_SHARED refers to the team of ...`